packages/bueno/rollup.config.js

```diff
@@ -23,46 +26,25 @@ function onWarn(warning, warn) {
   warn(warning);
 }
 
-const nodeConfig = {
-  input: 'src/index.ts',
-  output: [
-    {file: 'dist/bueno.js', format: 'cjs'},
-    {file: 'dist/bueno.esm.js', format: 'es'},
-  ],
-  plugins: [
-    resolve({preferBuiltins: true}),
-    commonjs(),
-    typescript(),
-    replace(),
-  ],
-  onwarn: onWarn,
-};
-
 const browserConfig = {
   input: 'src/index.ts',
-  output: [
-    {
-      file: 'dist/browser/bueno.js',
-      format: 'umd',
-      name: 'Bueno',
-      sourcemap: true,
-    },
-    {
-      file: 'dist/browser/bueno.esm.js',
-      format: 'es',
-      sourcemap: true,
-    },
-  ],
+  output: {
+    file: 'dist/browser/bueno.js',
+    format: 'umd',
```

Comment:
Since bueno is not being served on CDN,  a UMD bundle should in theory not be necessary.

I kept the bundle to remove any risk of a breaking change, but I think ESM should be enough since consumers of the package must be using a bundler?

Removing UMD support would allow us to remove the rollup config. The motivation would be to simplify the config to a single bundler; the time savings are small in this case.

With `@coveo/auth`, I opted to not provide a UMD as an experiment. We can decide to do the same with bueno at a later stage depending on how it goes,

Reply:
We can also do a major of bueno if that's an issue, I don't think it's that used except internally

Reply:
Since we now have a work-around to generate UMDs with esbuild, perhaps the best approach is to keep the UMD to minimize the chance of a breaking change, while still removing the rollup configuration. I will go with this 🤞